.flake8

```diff
@@ -2,6 +2,7 @@
 ignore = 
     ### Disabled because of conflicts with black
     E203, # whitespace before ':'
+	F841, # local variable is assigned but not never used
```

Comment:
E203 Has a comment in front of it justifying why it the test is disabled generally.

You should do the same for F841.

Right now I wonder two things:
1. Why did you not just delete the unreferenced variable
2. Why did you have to disable the warning globally and not only for the offending code.
 * Why did you globally

Reply:
I will have the comment above F841. @trym-b has mentioned it as well.
1. F.ex. one of the sample codes reads a zdf file, exports the point cloud, and then creates numpy arrays for xyz and rgb images. These variables (xyz and rgb) are not used later in the code. The point is for a user to have them available.
2. I thought that the sample code would not look nice.